src/Chainweb/Version.hs

```diff
@@ -750,20 +750,17 @@ oldTargetGuard _ _ = False
 
 -- | Skip validation of feature flags.
 --
--- Unused feature flag bits are supposed to be set to 0. This isn't enforced
--- currently (chainweb-node versions <= 1.6). There is a large number of blocks
--- in the history of mainnet before 2020-02-20, that have non-zero feature
--- flags.
---
--- This guard permits the use of the last 64 bits of a block header as a nonce
--- value for all blogs which pass this guard.
+-- Unused feature flag bits are supposed to be set to 0. As of Chainweb 1.7, the
+-- Feature Flag bytes and Nonce bytes have switched places in `BlockHeader`. For
+-- live chains, enforcing the following condition must be ignored for the
+-- historical blocks for which both the Nonce and Flags could be anything.
 --
 skipFeatureFlagValidationGuard
     :: ChainwebVersion
     -> BlockHeight
         -- ^ height of header
     -> Bool
-skipFeatureFlagValidationGuard Mainnet01 _ = True
+skipFeatureFlagValidationGuard Mainnet01 _ = False
 skipFeatureFlagValidationGuard Development _ = True
```

Comment:
For testing purposes, will probably want to set this to `True` only up to a certain block height, say, 2000.

Reply:
For which `ChainwebVersion`?

Reply:
Devnet, but we can still do it later when we reset devnet the next time.